tools_docker/test_docker/README.md

```diff
@@ -0,0 +1,45 @@
+# Docker Tests
+
```

Comment:
Missing information about:
- how to build this image `DOCKER_TRACIM_IMAGE=algoo/tracim:test`
- how to build this image `DOCKER_ELASTICSEARCH_IMAGE=elasticsearch-ingest:latest`
- why use this old version `DOCKER_COLLABORA_IMAGE=collabora/code:4.0.5.2`

Reply:
i will link to elasticsearch_ingest and tracim image building dock.
The version of collabora was not updated.

Reply:
4.0.5.2 is the only collabora version we can see in tracim documentation (it's also visible in setup collabora feature part backend doc), can you propose me a working more up to date version that in can put in documentation ?

Reply:
edit: version to use is : `collabora/code:4.2.6.2`

Reply:
done.